pom.xml

```diff
@@ -175,6 +178,11 @@
         <artifactId>metrics-ganglia</artifactId>
         <version>${shaded.dropwizard.ganglia.version}</version>
       </dependency>
+      <dependency>
+        <groupId>io.perfmark</groupId>
+        <artifactId>perfmark-api</artifactId>
+        <version>${shaded.perfmark.version}</version>
+      </dependency>
```

Comment:
The io.perfmark dependency comes with gRPC so that we don't have add it here.

Reply:
done